hack/load-tests/run-load-test.sh

```diff
@@ -233,15 +233,15 @@ function cleanup() {
 
 # shellcheck disable=SC2112
 function get_result_and_cleanup_trace() {
-    RECEIVED=$(curl -fs --data-urlencode 'query=round(avg(sum(rate(otelcol_receiver_accepted_spans{service="telemetry-trace-collector-metrics"}[20m]))))' localhost:9090/api/v1/query | jq -r '.data.result[] | .value[1]')
+    RECEIVED=$(curl -fs --data-urlencode 'query=round(avg(sum(rate(otelcol_receiver_accepted_spans{service="telemetry-trace-collector-metrics"}[1m]))))' localhost:9090/api/v1/query | jq -r '.data.result[] | .value[1]')
```

Comment:
The rate method returns a value per minute, which already reflects the average increase. Then you apply sum on it which sums up all entries in this minute vector.
The next avg seems redundant as you try to aggregate something which already was aggregated, nothing left to group.
And, you still have a a value per minute in your result, so which value you will take now? So either switch back to a 20min window or call an aggregate_over_time function to get a single value

Reply:
queries uodated